Assets/Mirror/Editor/Weaver/Processors/RpcProcessor.cs

```diff
@@ -81,16 +87,19 @@ public static MethodDefinition ProcessRpcCall(TypeDefinition td, MethodDefinitio
                 rpcName = rpcName.Substring(RpcPrefix.Length);
             }
 
-            // invoke SendInternal and return
-            // this
+            int channel = clientRpcAttr.GetField("channel", 0);
+            bool excludeOwner = clientRpcAttr.GetField("excludeOwner", false);
+
+            // invoke SendInternal and return this
```

Comment:
should be on 2 lines, the `// this` on the 2nd line is referring to `Ldarg_0`

Reply:
Please write a better comment as a suggestion and I'll accept it.  "this" on a blank line wasn't obvious enough to not appear as a line break in the sentence.